stdlib/asm/word.masm

```diff
@@ -12,6 +12,40 @@ export.reverse
     reversew
 end
 
+# CONVERSION UTILITIES
+#! Writes the components of a word to memory as eight u32 limbs in little-endian order.
+#!
+#! Inputs:  [W3, W2, W1, W0, OUT_PTR, ...]
+#! Outputs: [...]
+#!
+#! Where:
+#! - `W*` are the felts of the input word. Each felt is split into a low and high 32-bit limb.
+#! - `OUT_PTR` is an element address in memory.
+#! - Memory layout after the call: `[w0_lo, w0_hi, w1_lo, w1_hi, w2_lo, w2_hi, w3_lo, w3_hi]`.
+#!
+#! Cycles: 8 * (split + store pair) ~= 176
+export.store_word_u32s_le
+    reversew
+    # => [W0, W1, W2, W3, OUT_PTR, ...]
+
+    u32split
+    # => [W0_hi, W0_lo, W1, W2, W3, OUT_PTR, ...]
+
+    movup.2 u32split
+    # => [W1_hi, W1_lo, W0_hi, W0_lo, W2, W3, OUT_PTR, ...]
```

Comment:
nit: we could also store the first word to memory here and then drop it from the stack (rather than splitting all elements and then storing all of them after that). May simplify things a little.